projects/m1/012-what-color-is-that-square/js/index.js

```diff
@@ -0,0 +1,12 @@
+let letter = prompt("Choose a square")
+let number = (letter.slice(1))
+letter = (letter.slice(0, 1))
+
+ 
+if (letter === "B" || letter === "D" || letter === "F" || letter === "H") { number = number - 1 }
+else if (letter === "A" || letter === "C" || letter === "E" || letter === "G") 
+{ number = number - 0 }
```

Comment:
tutto questo `else if` che senso ha? che senso ha un risultato `number = number - 0`? Se togliessi interamente riga 7 e 8 cosa cambierebbe nel codice?
Attenzione a riflettere sempre se un controllo è necessario o meno, perché si tratta pur sempre di uno "sforzo" in più che chiedi al software